<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:rgb(85, 170, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">QTabWidget QTabBar{
border-top: 2px solid #C2C7CB;
width: 256px;
font: 10pt &quot;Arial&quot;;
background-color:rgb(255, 0, 255);
}

QTabWidget QTabBar::tab{
background-color: #136ba2;
height: 30px;
width: 110px;
color: #e6e6e6;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Ploting</string>
          </attribute>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Visual</string>
          </attribute>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>30</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Broswer..</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>91</y>
             <width>231</width>
             <height>31</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Pi Chart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bar Chart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scatter Chart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Line Chart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spline Chart</string>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="comboBox_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>171</y>
             <width>231</width>
             <height>31</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Light</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue Cerulean</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dark</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Brown Sand</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue NCS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High Contrast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue Icy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Qt</string>
            </property>
           </item>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCustomPlot" name="plot" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="Visualization" name="visual" native="true">
      <property name="styleSheet">
       <string notr="true">#visual{
border-top: 2px solid #C2C7CB;
width: 256px;
font: 10pt &quot;Arial&quot;;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Visualization</class>
   <extends>QWidget</extends>
   <header location="global">visualization.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
